external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -442,6 +442,8 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSQLCo
       .option("lowerBound", "2018-07-06")
       .option("upperBound", "2018-07-20")
       .option("numPartitions", 3)
+      .option("oracle.jdbc.mapDateToTimestamp", "false")
```

Comment:
This is good to know. Could you help update the doc and explain we can pass the options to the underlying JDBC and also give some examples?

Reply:
Also leave comments before line 445 to explain why we need to do this.

Reply:
ok. I will add notes to http://spark.apache.org/docs/latest/sql-programming-guide.html#jdbc-to-other-databases, and will also add comments to the code.